benchmarks-mobile/protolite_app/app/src/main/java/io/grpc/grpcbenchmarks/ProtobufBenchmarker.java

```diff
@@ -128,8 +155,8 @@ public static BenchmarkResult serializeJsonToByteArray(final String jsonString,
                 @Override
                 public void execute() throws IOException {
                     ByteArrayOutputStream bos = new ByteArrayOutputStream(serializedSize);
-                    GZIPOutputStream gos = new GZIPOutputStream(bos);
-                    gos.write(jsonObject.toString().getBytes());
+                    BestGZIPOutputStream gos = new BestGZIPOutputStream(bos);
+                    gos.write(jsonObject.toString().getBytes("UTF-8"));
```

Comment:
Seems like StandardCharsets wasn't added until API 19 for Android, the way to do this is still use the name.